bin/varnishtest/tests/b00092.vtc

```diff
@@ -0,0 +1,28 @@
hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world&hello=world"
```

Comment:
I'll be glad if someone comes up with a clean way to write this kind of test :)

Reply:
We have `${string,repeat,4096,x}`but dunno if that works.

Reply:
Replace `x` with `&hello=world` and it should :)

Reply:
maybe reduce `http_req_size` to the minimum can help too

Reply:
Alright I'll try all that

Reply:
Thx for the tips guys !